tutorial/src/App.js

```diff
@@ -12,52 +14,96 @@ import TqecApp from './workspace'
  *  with the web app: https://ivanceras.github.io/svgbob-editor/
  */
 
+const handleTabClick = (tabId) => {
```

Comment:
We need to come to a consensus on how we want to design the different tabs.

Reply:
I had to choose an implementation to move the prototype forward.
I am happy to discuss alternative implementations.